configure-nats.sh

```diff
@@ -5,16 +5,18 @@
 
```

Comment:
We're not using this file yet, but it's updated to work with the latest connect.